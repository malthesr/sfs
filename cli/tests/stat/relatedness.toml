args = [
  "stat",
  "-s",
  "r0,r1,king",
  "--header",
  "tests/stat/two_individuals.sfs"
]