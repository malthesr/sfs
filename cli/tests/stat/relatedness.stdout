r0,r1,king
1.500000,0.068966,-0.133333
